lib/lighthouse/benefits_documents/service.rb

```diff
@@ -34,9 +34,10 @@ def queue_document_upload(params, lighthouse_client_id = nil)
     end
 
     def queue_multi_image_upload_document(params, lighthouse_client_id = nil)
-      loggable_params = filter_sensitive_params(params)
-      loggable_params[:icn] = @user.icn
```

Comment:
I don't believe we should be logging the user's ICN, since it's now considered PII.